_P178_SCD30.ino

```diff
@@ -142,6 +149,12 @@ boolean Plugin_178(byte function, struct EventStruct *event, String& string)
           success = true;
           scd30.getAltitudeCompensation(&value);
           SendStatus(event->Source, "Altitude: "+String(value,DEC));
+        } else if (string.equalsIgnoreCase(F("SCDGETTMP")))
+        {
+          success = true;
+          float temp;
+          scd30.getTemperatureOffset(&temp);
+          SendStatus(event->Source, "Temp offset: "+String(temp,2));
```

Comment:
These strings `"Temp offset: "` should also be wrapped in F() macro.
You probably cannot use these with simple string concatenation so you must very likely declare a separate String object for it.

The thing with these strings (witthout using the F() macro) is that they use a lot of memory (RAM) when not marked with a marker to put them in flash.

Reply:
Fixed